src/components/calendar/calendar.jsx

```diff
@@ -188,6 +303,7 @@ function Calendar(props) {
 
 Calendar.propTypes = {
   value: PropTypes.string,
+  refs: PropTypes.Object,
```

Comment:
This looks like it is not used. And probably throwing linter violation errors since we do not have a default for it.